### ISSUE TITLE
can't find "from sklearn.cross_validation import train_test_split" in Latest version scikit-learn

### ISSUE THREAD

**Issue by Stoneww** [Avik-Jain/100-Days-Of-ML-Code#53]
**Describe the bug**
can't find "from sklearn.cross_validation import train_test_split" in Latest version scikit-learn
**To Reproduce**
- Day1 
- Step 5: Splitting the datasets into training sets and Test sets
- Can't find "from sklearn.cross_validation import train_test_split" in Latest version scikit-learn**

**Desktop (please complete the following information):**
 - OS: win10
 - Browser chrome

**Additional context**

- Is there any other way to replace this module?
THANKS

**Comment by jackycsie** [comment]
Hello 

You can use this code
from sklearn.model_selection import train_test_split

this module can split train set and test set
X_train, X_test, Y_train, Y_test = train_test_split(X, Y, test_size = 0.25, random_state = 0)
